docs/app/pages/components/components-sections/notifications/badge/badge.component.html

```diff
@@ -0,0 +1,158 @@
+                        </div>
+                    </div>
+                </div>
+                <div class="row uxd-customize-row">
+                    <div class="col-md-3 col-sm-6">
+                        <label for="buttonSize">badgeVerticalPosition</label>
+                    </div>
+                    <div class="col-md-3 col-sm-6">
+                        <ux-radio-button [(value)]="badgeVerticalPosition" option="above">above</ux-radio-button>
+                        <ux-radio-button [(value)]="badgeVerticalPosition" option="below">below</ux-radio-button>
+                    </div>
+                    <div class="col-md-3 col-sm-6">
+                        <label for="buttonSize">badgeHorizontalPosition</label>
+                    </div>
+                    <div class="col-md-3 col-sm-6">
+                        <ux-radio-button [(value)]="badgeHorizontalPosition" option="before">before</ux-radio-button>
+                        <ux-radio-button [(value)]="badgeHorizontalPosition" option="after">after</ux-radio-button>
+                    </div>
+                </div>
+                <div class="row uxd-customize-row">
+                    <div class="col-md-2 col-sm-3">
+                        <label for="buttonSize">badgeSize</label>
+                    </div>
+                    <div class="col-md-4 col-sm-9">
+                        <ux-radio-button [(value)]="badgeSize" option="small">small</ux-radio-button>
+                        <ux-radio-button [(value)]="badgeSize" option="medium">medium</ux-radio-button>
+                        <ux-radio-button [(value)]="badgeSize" option="large">large</ux-radio-button>
+                    </div>
+                    <div class="col-md-3 col-sm-6">
+                        <ux-checkbox [(value)]="badgeOverlap">badgeOverlap</ux-checkbox>
+                    </div>
+                    <div class="col-md-3 col-sm-6">
+                        <ux-checkbox [(value)]="badgeHidden">badgeHidden</ux-checkbox>
+                    </div>
+                </div>
+            </ux-accordion-panel>
+        </ux-accordion>
+    </div>
+</div>
+
+<hr />
+
+<p>
+    The <code>uxBadge</code> directive can be added to any element to provide additional information as a badge, such as
+    status text or unread count.
+</p>
+<p><strong>Note:</strong> the badge directive relies on the CSS overflow property being set to <code>visible</code> for
+    the subject element due to how the badge element attaches to it. If you want to use <code>hidden</code>, you will
+    need to add a wrapper element around the subject so it only affects the contents.</p>
+<p>
+    The appearance and behavior of the directive can be configured using the following properties:
+</p>
+
+<uxd-api-properties tableTitle="Inputs">
+    <tr uxd-api-property name="badgeColor" type="string">
+        The color you want to use for the background color. This should be a string value that matches one of the colors
```

Comment:
Do these colour inputs also support CSS colour values too? If so we should probably mention that.